### PR TITLE
Fix nxp 26150 comment submit shortcut

### PR DESCRIPTION
**Pull request by nmpcunha**

### CODE REVIEW

File: elements/nuxeo-document-comments/nuxeo-document-comment-thread.html [hunk 1]
```diff
@@ -352,6 +357,9 @@
       },
 
       /** Visibility Methods **/
+      _allowReplies: function (level) {
```
Comment (Gabez0r) [inline elements/nuxeo-document-comments/nuxeo-document-comment-thread.html]: white space between function and parmeters:
```suggestion
      _allowReplies: function(level) {
```

File: elements/nuxeo-document-comments/nuxeo-document-comment-thread.html [hunk 2]
```diff
@@ -286,7 +288,10 @@
         this._fetchComments(this.allCommentsLoaded);
       },
 
-      _submitReply: function() {
+      _submitReply: function(e) {
+        if(e) {
```
Comment (Gabez0r) [inline elements/nuxeo-document-comments/nuxeo-document-comment-thread.html]: missing space between if and condition:
```suggestion
        if (e) {
```

**Comment by Gabez0r** [comment]
👍

**Comment by guirenard** [comment]
:+1:

**Pull request merged**